pyodide-build/pyodide_build/install_xbuildenv.py

```diff
@@ -15,12 +15,27 @@ def make_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         "on numpy or scipy.\n"
         "Note: this is a private endpoint that should not be used outside of the Pyodide Makefile."
     )
+    parser.add_argument("--download", action="store_true", help="Download xbuild env")
     parser.add_argument("xbuild_env", type=str, nargs=1)
     return parser
 
 
-def main(args: argparse.Namespace) -> None:
-    xbuildenv_path = Path(args.xbuild_env[0])
+def download_xbuild_env(xbuildenv_path: Path) -> None:
+    from shutil import rmtree, unpack_archive
+    from tempfile import NamedTemporaryFile
+    from urllib.request import urlretrieve
+
+    rmtree(xbuildenv_path, ignore_errors=True)
+    with NamedTemporaryFile(suffix=".tar") as f:
+        urlretrieve(
+            "http://pyodide-cache.s3-website-us-east-1.amazonaws.com/xbuildenv/1.tar",
```

Comment:
I'm intending to deploy these to e.g., `xbuildenv/0.21.0a3.tar". So in the future we will look up the version of `pyodide_build` and use that in the string here.